spec/features/user_views_a_showpage_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+require 'pry'
+
+feature 'View one dev tool on a unique show page' do
+  let!(:devtool1) {FactoryGirl.create(:devtool, id: 1)}
+  let!(:review1) {FactoryGirl.create(:review, devtool_id: 1)}
+  let!(:devtool2) {FactoryGirl.create(:devtool, id: 2)}
+  let!(:review2) {FactoryGirl.create(:review, devtool_id: 2)}
```

Comment:
let!(:review2) {FactoryGirl.create(:review, devtool_id: 2)} should be:

let!(:review2) {FactoryGirl.create(:review, devtool: devtool2)}

This makes the tests less brittle